staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2446,6 +2446,22 @@ type Container struct {
 	// +optional
 	// +listType=atomic
 	ResizePolicy []ContainerResizePolicy `json:"resizePolicy,omitempty" protobuf:"bytes,23,rep,name=resizePolicy"`
+	// Restart policy for the container.
+	// This may only be set for init containers.
+	// The only supported value is "Always".
+	// When set to "Always", this init container will continuously restart if it
+	// exits for any reason, until all the regular containers have terminated.
+	// This init container will be terminated only after all the regular
+	// containers have terminated. This lifecycle differs from normal init
+	// containers and is often referred to as a "sidecar" container. Although
```

Comment:
Suggested rewording:

“An init container that uses Always is considered a “sidecar” - it does not impact the lifecycle of the pod but instead supports other containers”

or similar.

Reply:
I kept as-is, but rephrased a bit around. Should be better now